core/domain/wipeout_domain.py

```diff
@@ -56,8 +56,12 @@ def create_default(cls, user_id, email, exploration_ids, collection_ids):
         Args:
             user_id: str. The ID of the user who is being deleted.
             email: str. The email of the user who is being deleted.
-            exploration_ids: list(str).
-            collection_ids: list(str).
+            exploration_ids: list(str). Private explorations that are marked as
+                deleted and need to be hard deleted. These are known at the time
+                when user requests deletion so we can set them outright.
```

Comment:
... deletion, so we can set them at the outset.

Ditto below.

Also, "hard deleted" --> "hard-deleted" (in both places).